nginx/nginx_test.go

```diff
@@ -311,6 +310,7 @@ func TestNginxConfigUpdates(t *testing.T) {
 					ServiceAddress: "anotherservice",
 					ServicePort:    6060,
 					Allow:          []string{"10.86.0.0/16"},
+					StripPaths:     false,
```

Comment:
this has moved to the controller. It is always set there starting with the default and only overidden of the annotation exists